css/utils/vars.css

```diff
@@ -21,6 +21,7 @@
   --badge-color: #e67a00;
   --text-green: green;
```

Comment:
Avoid using direct color names and use hex/rgb format as different browsers might different values assigned to it